### PR TITLE
lolcat 42.24.0 (new formula)

### PR DESCRIPTION
**Pull request by cglong**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/lolcat.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class Lolcat < Formula
+  desc "Rainbows and unicorns in your console!"
+  homepage "https://github.com/busyloop/lolcat"
+  url "https://github.com/busyloop/lolcat.git", :using => :git, :tag => "v42.24.0", :revision => "dfc68649f6bdac255d5be052d2123f3fbe3f555c"
```
Comment (ilovezfs) [inline Formula/lolcat.rb]:
```
  url "https://github.com/busyloop/lolcat.git",
      :tag => "v42.24.0",
      :revision => "dfc68649f6bdac255d5be052d2123f3fbe3f555c"
```

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
@cglong Thanks :unicorn:

**Comment by cglong** [comment]
Haha, thanks @ilovezfs 😄

**Comment by ilovezfs** [comment]
You're welcome :)

**Pull request closed without merging**